packages/dashboard/src/vaadin-dashboard-widget.js

```diff
@@ -68,10 +68,19 @@ import { DashboardSection } from './vaadin-dashboard-section.js';
  *
  * The following custom properties are available:
  *
- * Custom Property                   | Description
- * ----------------------------------|-------------
- * `--vaadin-dashboard-item-colspan` | colspan of the widget
- * `--vaadin-dashboard-item-rowspan` | rowspan of the widget
+ * Custom Property                             | Description
+ * --------------------------------------------|----------------------
+ * `--vaadin-dashboard-item-colspan`           | colspan of the widget
```

Comment:
Should we align the naming for all the properties to maybe use `--vaadin-dashboard-item-xyz`?

Reply:
Or, rather, renamed `-item-` properties to `-widget-`, since they're all widgets anyway